Firebase.PerformanceMonitoring/source/Firebase.PerformanceMonitoring/Firebase.PerformanceMonitoring.targets

```diff
@@ -2,11 +2,16 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
 		<_FirebasePerformanceMonitoringAssemblyName>Firebase.PerformanceMonitoring, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null</_FirebasePerformanceMonitoringAssemblyName>
-		<_FirebasePerformanceMonitoringItemsFolder>FPrfrmncM-1.0.6</_FirebasePerformanceMonitoringItemsFolder>
+		<_FirebasePerformanceMonitoringItemsFolder>FPrfrmncM-1.1.0</_FirebasePerformanceMonitoringItemsFolder>
+		<_FirebaseSwizzlingUtilitiesItemsFolder>FSWU-1.0.0</_FirebaseSwizzlingUtilitiesItemsFolder>
```

Comment:
I would suggest changing this to "FSwzzlngU" to keep the same naming with the others ItemsFolder names.